examples/lib/lib_test.go

```diff
@@ -24,3 +24,7 @@ func TestMeaning(t *testing.T) {
 		t.Errorf("got %d, want 42", m)
 	}
 }
+
+// func TestAddTwoNumbers(t *testing.T) {
```

Comment:
I commented out this test because it fails. I need bazel team's help to make it work.